generator/src/components/Button/index.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+  export let onClick: () => void
+  export let icon: string | null = null
+  export let title: string
+</script>
+
+<button class="button" on:click={onClick}>
+  {#if icon}
+    <img class="icon" src={icon} alt="icon" />
+  {/if}
+  {#if title}
+    <span class="title">{title}</span>
```

Comment:
Do we want a button without a title?

Reply:
I don't remember seeing any buttons without a title in the designs, can you please point them out?

Reply:
Ok, title is not optional now :-)